presentation/src/main/java/com/anytypeio/anytype/presentation/widgets/WidgetConfig.kt

```diff
@@ -51,7 +51,7 @@ object WidgetConfig {
 object BundledWidgetSourceIds {
     const val FAVORITE = "favorite"
     const val RECENT = "recent"
-    const val RECENT_LOCAL = "recent_local"
+    const val RECENT_LOCAL = "recentOpen"
```

Comment:
let's rename it to be in sync with the value: `const val RECENT_OPEN`

Reply:
I don't like the naming copied from the desktop